stories/Display/TabsStory.tsx

```diff
@@ -15,28 +19,91 @@ const props = {
       require: 'yes',
       description: 'Sets title of Tab',
     },
+    {
+      name: 'alignment',
+      type: 'string',
+      defaultValue: 'horizontal',
+      possibleValue: 'horizontal, vertical',
+      require: 'yes',
+      description: 'Sets alignment of Tab',
```

Comment:
Can you also explain here that the vertical tabs will only show at the appropriate break point `768px`